apps/testingaccessibility/src/utils/send-server-email.ts

```diff
@@ -59,11 +65,12 @@ export async function sendServerEmail({
   const params = new URLSearchParams({callbackUrl, token, email: identifier})
   const _url = `${baseUrl}/api/auth/callback/${emailProvider.id}?${params}`
 
-  await emailProvider.sendVerificationRequest({
```

Comment:
this always used the default send function so instead we export it so we are using our custom send